Examples/Framework/include/ActsExamples/EventData/ModuleCluster.hpp

```diff
@@ -0,0 +1,38 @@
+// This file is part of the Acts project.
+//
+// Copyright (C) 2021 CERN for the benefit of the Acts project
+//
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+#pragma once
+
+#include "Acts/Digitization/Segmentation.hpp"
+#include "ActsExamples/EventData/IndexSourceLink.hpp"
+#include "ActsExamples/EventData/Measurement.hpp"
+
+#include <cassert>
+#include <vector>
+
+namespace ActsExamples {
+template <typename measurement_t>
+class ModuleCluster {
```

Comment:
Is the only need for this to forward the module segmentation ?
We could probably do this via the GeometryIdentifer ...

Reply:
Are you suggesting using Measurement as input for both pixel and strip SP builders, retrieving the module segmentation via GeometryIdentifier in the strip SP builder?